mne/utils.py

```diff
@@ -1370,23 +1325,75 @@ def _chunk_read_ftp_resume(url, temp_file_name, local_file, verbose_bool=True):
     if len(server_path) > 1:
         data.cwd(unquoted_server_path)
     data.sendcmd("TYPE I")
-    data.sendcmd("REST " + str(local_file_size))
+    data.sendcmd("REST " + str(initial_size))
     down_cmd = "RETR " + file_name
-    file_size = data.size(file_name)
-    progress = ProgressBar(file_size, initial_value=local_file_size,
-                           max_chars=40, spinner=True, mesg='downloading',
+    assert file_size == data.size(file_name)
```

Comment:
`assert ` statements have no guarantee of being evaluated because they can be skipped by `python -O` (or something like that), so it's not a good idea to use them if you think the condition should ever be violated. I use them in places where I want to remind myself that a condition should hold based on what I've coded, and also get some sanity checks during development.